tasks/bower-requirejs.js

```diff
@@ -1,18 +1,56 @@
 'use strict';
 var bowerRequireJS = require('bower-requirejs');
+var path = require('path');
+var multimatch = require('multimatch');
+var findup = require('findup-sync');
+
+/**
+ * Task
+ */
+function task() {
+  var options = this.options({
+    config: this.data.rjsConfig,
+    exclude: [],
+    baseUrl: '',
+    transitive: false,
+    excludeDev: false
+  });
+
+  options['exclude-dev'] = options.excludeDev;
+
+  bowerRequireJS(options, this.async());
+}
+
+/**
+ * Check package.json for installed bower task
+ * @returns {boolean}
+ */
+function isBowerTaskInstalled() {
+  var config = findup('package.json');
+  var scope = ['dependencies', 'devDependencies', 'peerDependencies'];
+
+  if (typeof config === 'string') {
+    config = require(path.resolve(config));
+  }
+
+  var names = scope.reduce(function (result, prop) {
+    return result.concat(Object.keys(config[prop] || {}));
+  }, []);
+
+  return !!multimatch(names,['!grunt-bower-requirejs','grunt-bower*']).length;
```

Comment:
@sindresorhus do you think testing the module names is enough or should i load the matching tasks before `grunt.task.exists('bower')` to make sure the found module registers a conflicting `bower` task?
This one should be faster.